include/deal.II/fe/fe_p1nc.h

```diff
@@ -301,8 +301,8 @@ class FE_P1NC : public FiniteElement<2, 2>
    * Do the work which is needed before cellwise data computation.
    * Since the shape functions are constructed independently on each cell,
    * the data on the reference cell is not necessary.
-   * It returns an empty variable type of @ InternalDataBase and updates @
-   * update_flags, and computes trivially zero Hessian for each cell if it is
+   * It returns an empty variable type of @ InternalDataBase and updates
```

Comment:
```suggestion
   * It returns an empty variable of type InternalDataBase and stores
```
This sentence is a bit odd. I suggest to move `type` to the `@` symbol. Plus this function does not update the `update_flags` parameter, but rather stores them: It updates the internal `update_flags` based on the provided `update_flags`. See also the implementation [here](https://www.dealii.org/developer/doxygen/deal.II/fe__p1nc_8cc_source.html#l00136).

Reply:
Thanks for actually reading the text. I just rotely changed the places with a trailing, unaccompanied `@` :-)

So fixed.